src/components/tx/TxData.js

```diff
@@ -411,7 +427,7 @@ class TxData extends React.Component {
           try {
             script = atob(output.script);
           } catch (e) {
-            console.error(e);
+            console.log(e);
```

Comment:
chore: We should keep the `error()` method here, as this is an error parsing the script data.

Reply:
Hi, method corrected. Waiting for review.🙂